adafruit_thermal_printer/thermal_printer.py

```diff
@@ -117,13 +124,17 @@ class _PrintModeBit:
         # which by design only implements get, set, init.  As a result workaround
         # this pylint issue by disabling the warning.
         # pylint: disable=too-few-public-methods
-        def __init__(self, mask):
+        def __init__(self, mask: int) -> None:
             self._mask = mask
 
-        def __get__(self, obj, objtype):
+        def __get__(
+            self,
+            obj: "ThermalPrinter",
```

Comment:
`obj` can be `None` if it's access as a class attribute, so this should be wrapped with `Optional`.

Reply:
oops.  yes.  Just like my notes on those types of objects says it should be..... [ dang! ]